bip-0003.md

```diff
@@ -0,0 +1,753 @@
+* Apache-2.0: [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+* BSL-1.0: [Boost Software License, version 1.0](http://www.boost.org/LICENSE_1_0.txt)
+
+#### Not Acceptable Licenses
+
+All licenses not explicitly included in the above lists are not acceptable terms for a Bitcoin Improvement Proposal.
+However, BIPs predating the acceptance of this BIP were allowed under other terms, and should use these abbreviations
+when no other license is granted:
+
+* PD: Released into the public domain
+* OPL: [Open Publication License, version 1.0](http://opencontent.org/openpub/)
+
+## BIP Editors
+
+The current BIP editors are:
+
+* Bryan Bishop ([kanzure@gmail.com](mailto:kanzure@gmail.com))
+* Jon Atack ([jon@atack.com](mailto:jon@atack.com))
+* Luke Dashjr ([luke_bipeditor@dashjr.org](mailto:luke_bipeditor@dashjr.org))
+* Mark "Murch" Erhardt ([murch@murch.one](mailto:murch@murch.one))
+* Olaoluwa Osuntokun ([laolu32@gmail.com](mailto:laolu32@gmail.com))
+* Ruben Somsen ([rsomsen@gmail.com](mailto:rsomsen@gmail.com))
+
+### BIP Editor Responsibilities and Workflow
+
+The BIP editors subscribe to the Bitcoin Development Mailing List and watch the [BIPs
+repository](https://github.com/bitcoin/bips).
+
+When a new BIP idea is submitted to the mailing list, BIP editors and other community members should comment in regard
+to:
+
+* Novelty of the idea
+* Viability, utility, and relevance of the concept
+* Readiness of the proposal
+* On-topic for the Bitcoin community
+
+Discussion in pull request comments can often be hard to follow as feedback gets marked as resolved when it is addressed
+by authors. Substantive discussion of ideas may be more accessible to a broader audience on the mailing list, where it
+is also more likely to be retained by the community memory.
+
+If the BIP needs more work, an editor should ensure that constructive, actionable feedback is provided to the authors
+for revision. Once the BIP is ready it should be submitted as a "pull request" to the [BIPs
+repository](https://github.com/bitcoin/bips) where it may get further feedback.
+
+For each new BIP pull request that comes in, an editor checks the following:
+
+* The idea has been previously discussed on the Bitcoin Development Mailing List
+* Title accurately describes the content
+* Proposal is of general interest and/or pertains to more than one Bitcoin project/implementation
+* Document is properly formatted
+* Licensing terms are acceptable
+* Motivation, Rationale, and Backward Compatibility have been addressed
+* Specification provides sufficient detail for implementation
+* The defined Layer header must be correctly assigned for the given specification
+* The BIP is ready: it is comprehensible, technically feasible, and all aspects are addressed as necessary
+
+Editors do NOT evaluate whether the proposal is likely to be adopted.
+
+A BIP editor will:
+
+* Assign a BIP number and BIP type in the pull request
```

Comment:
I mildly wonder if people could use this section to argue that an editor must assign a number to a proposal that fulfills the list above while yet considered out of scope of the BIPs repository.

Reply:
Good point, I added "The described idea is on-topic for the repository" after the first point, and added a "Then, " before "a BIP Editor will".